calculator.rb

```diff
@@ -0,0 +1,34 @@
+def prompt_operator
+  puts "enter a math operation"
+  operator = gets.chomp
+end
+
+def prompt_number
+  begin
+    puts "enter a number for the math operation"
+    Float(gets.chomp) 
+  rescue ArgumentError
+    puts "invalid number"
+    retry
+  end
```

Comment:
Just as a heads up, we will get deeper into `begin ... rescue` clauses a few weeks from now!